File: app/src/main/java/com/example/login/MainActivity.java
package com.example.login;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);




        getSupportActionBar().hide();


    }

    public void click (View v){
        Intent intent;
        switch (v.getId()){
            case R.id.signin1:
                intent = new Intent(MainActivity.this,signup.class);
                break;
            case R.id.signin2: // R.id.textView2
                intent = new Intent(this, signup.class);
                break;
        }
       startActivity(intent);

    }
}

File: app/src/main/java/com/example/login/MainActivity.kt
package com.example.login

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)




        supportActionBar!!.hide()


    }

    fun click(v: View) {
        val intent: Intent
        when (v.id) {
            R.id.signin1 -> intent = Intent(this@MainActivity, signupo)
            R.id.signin2 // R.id.textView2
            -> intent = Intent(this, signup)
        }
        startActivity(intent)

    }
}
